### ISSUE TITLE
 If block start with a parenthesis after the `if`, the indentation disappears.

### ISSUE THREAD

**Issue by zmunm** [pinterest/ktlint#829]
## Expected Behavior

context: https://github.com/detekt/detekt/issues/2964
Even if written without curly braces, indentation should be preserved

## Observed Behavior

 If block start with a parenthesis after the `if`, the indentation disappears.

## Steps to Reproduce

Before formatting

```kotlin
if (true)
    (1).toString() 
else
    2.toString()
```

After formatting

```kotlin
if (true)
(1).toString()  // indentation disappeared
else
    2.toString()
```

## Your Environment

* Version of ktlint used: 0.37.2
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): standard gradle dsl task from ktlint README
* Version of Gradle used (if applicable): 5.6.4
